app/kumactl/cmd/export/export.go

```diff
@@ -91,7 +85,11 @@ $ kumactl export --profile federation --format universal > policies.yaml
 				}
 				if resDesc.Scope == model.ScopeGlobal {
 					list := resDesc.NewList()
-					if err := rs.List(cmd.Context(), list); err != nil {
+					// filter out envoy-admin-ca and inter-cp-ca otherwise it will cause TLS handshake errors
+					if err := rs.List(cmd.Context(), list, store.ListByFilterFunc(func(rs model.Resource) bool {
```

Comment:
please double check that it works with remote store on a real system. Becuase I think it does not. I'd filter this after we retrieve the list

Reply:
this is how I checked:

```
build/artifacts-darwin-arm64/kumactl/kumactl export --format kubernetes | grep -i serviceinsight

```

```
build/artifacts-darwin-arm64/kumactl/kumactl export --format kubernetes --profile all | grep -i serviceinsight
type: ServiceInsight
type: ServiceInsight
type: ServiceInsight
```

Reply:
not it is filtered out 

```
build/artifacts-darwin-arm64/kumactl/kumactl export --format kubernetes | grep -i envoy-admin-ca

```